tests/test_utils.py

```diff
@@ -52,15 +71,35 @@ def test_path_normalized(self, mock_current_app, mock_isdir):
         assert mock_isdir.call_args[0][0].endswith(expected)
 
 
+class TestAssignValForSorting():
+    @pytest.mark.parametrize('item, expected_value', [
+        # Captions should show up first so it gets the 'a' prefix
+        ({'vtt_kind': 'captions', 'language': 'fre'}, 'afre'),
+        # English should sort before other languages, so replace with 'aaa'
+        ({'vtt_kind': 'thumbnails', 'language': 'eng'}, 'faaa'),
+        ({'vtt_kind': 'chapters', 'language': 'ger'}, 'cger'),
+        # Missing or unknown vtt kinds sort last
+        ({'language': 'spa'}, 'gspa'),
+        # Missing languages sort last
+        ({'vtt_kind': 'metadata'}, 'ezzz'),
+        ({}, 'gzzz'),
+    ])
+    def test_assigns_expected_values(self, item, expected_value):
+        actual_value = assign_val_for_sorting(item)
+        assert actual_value == expected_value
+
+
 @mock.patch('aubrey_transcription.utils.decrypt_filename')
 @mock.patch('aubrey_transcription.utils.current_app', config={
     'EXTENSIONS_META': {'vtt': {'mimetype': 'text/vtt', 'use': 'vtt'}},
     'PAIRTREE_BASE': '/some/path',
     'TRANSCRIPTION_URL': 'http://example.com',
 })
 class TestGetFilesInfo():
+    @mock.patch('aubrey_transcription.utils.assign_val_for_sorting')
     @mock.patch('aubrey_transcription.utils.os.path.getsize')
-    def test_returns_correct_info(self, mock_getsize, mock_current_app, mock_decrypt_filename):
+    def test_returns_correct_info(self, mock_getsize, mock_assign_val_for_sorting,
+            mock_current_app, mock_decrypt_filename):
```

Comment:
A small change per flake8 `tests/test_utils.py:102:13: E128 continuation line under-indented for visual indent`

Reply:
it will be fixed!